src/po/Make_all.mak

```diff
@@ -189,8 +189,8 @@ PO_VIM_INPUTLIST = \
 	../../runtime/defaults.vim
 
 PO_VIM_JSLIST = \
-	optwin.js \
-	defaults.js
+	________runtime__optwin.js \
+	________runtime__defaults.js
```

Comment:
why are those files renamed?

Reply:
I needed the directories that those files reside in, so that both a plugin/foo.js _and_ an autoload/foo.js can be read for messages to translate. Besides, these are temporary files...